web3js-ext/src/web3/web3.ts

```diff
@@ -51,6 +84,13 @@ export class KlaytnWeb3 extends Web3 {
     };
   }
 
+  accounts_recoverTransaction(context: Web3Context): typeof this.eth.accounts.recoverTransaction {
+    // signTransactionWithContext. see web3/src/accounts.ts:initAccountsForContext
+    return (rawTransaction: HexString): Address => {
+      return recoverTransactionWithKlaytnTx(context, rawTransaction);
+    };
+  }
+
   accounts_signTransactionAsFeePayer(context: Web3Context): typeof this.eth.accounts.signTransaction {
```

Comment:
Let's move this part to account.ts `signTransactionAsFeePayerWithContext`, just like other `~WithContext` functions.

Reply:
Good.
signTransactionAsFeePayerWithContext is moved into initAccountsForContext in https://github.com/klaytn/web3klaytn/pull/522/commits/6d6829b8daa26381fd7bc1fd7933b09a5f0e2414.